website/content/docs/sync/index.mdx

```diff
@@ -127,6 +127,24 @@ Granularity can be updated. The new granularity only affects secrets newly assoc
 not modify the previously synced secrets. It is possible to update an association to force a recreate operation.
 The secret synced with the old granularity will be deleted and new secrets will be synced according to the new granularity.
 
+
+## Reconciliation
+
+Vault Secrets Sync is designed to automatically recover from transient failures
+in two ways: operation retries and reconciliation scans.
+
+Operation retries happen when a sync operation fails. Vault automatically
+retries the operation with exponential backoff. Operation retries help in
+situations where your network becomes unreliable or overwhelmed.
+
+Reconciliation scans happen periodically in a background thread. Vault scans all
+secrets currently managed by the sync system to identify and update out-of-date
+secrets, and to ensure that any configured destinations are up-to-date.
+Reconciliation scans help in situations where you have dropped update events or
+external service downtimes that are outside of your control.
```

Comment:
I don't think I'd mention dropped events. I'm afraid it'll lead to uncomfortable questions and paint Vault as unreliable. I'd just mention it lets us automatically recover/self-heal from outstanding service disruptions or something and leave it at that. Put the emphasis on how it protects us against the whims of the 3rd party.